apps/e2e/src/engine-client/fullSanity.ts

```diff
@@ -437,6 +437,16 @@ async function fullSanity(context: RunContext) {
       );
     }
   });
+
+  const transferQuoteResult = await client.transferQuote({
+    chainId,
+    recipientSubaccountName: 'transfer1',
+    subaccountOwner: signer.address,
+    subaccountName: 'default',
+    amount: toFixedPoint(50), // amount must be x18
```

Comment:
Does this mean the decimals must be `18`? 

Because when trying with USDC (`6`) I get an error that the amount is below the min (which it shouldn't be). If I add `18` decimals to it though the transfer succeeds with the correct amount. Should I just always be adding `18`?

Reply:
Yeah, decimals should always be 18 (so you would just call `addDecimals` without the second argument)